extensions/funqy/funqy-knative-events/deployment/src/test/java/io/quarkus/funqy/test/ContextFunction.java

```diff
@@ -10,13 +10,13 @@
     public void context(String body, @Context CloudEvent event) {
         if (event == null)
             throw new RuntimeException();
-        if (!event.specVersion().equals("1.0"))
+        if (!event.specVersion().equals(CloudEvent.SpecVersion.V1))
             throw new RuntimeException();
         if (!event.id().equals("1234"))
             throw new RuntimeException();
         if (!event.subject().equals("bb"))
             throw new RuntimeException();
-        if (!event.source().equals("test"))
+        if (!event.source().toString().equals("test"))
```

Comment:
I believe this is no longer needed (the `toString`) as source is string anyway

Reply:
Yes you are correct.